packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -128,7 +122,8 @@ export default function(api, options) {
     inherits: syntaxOptionalChaining,
 
     visitor: {
-      "MemberExpression|CallExpression"(path) {
+      "OptionalCallExpression|OptionalMemberExpression"(path) {
+        debugger;//eslint-disable-line
```

Comment:
Remove.

Reply:
Sorry for that :sweat_smile: , test cases were passing and i completely forgot that lived here.